modules/fbx/fbx_document.cpp

```diff
@@ -2127,6 +2136,9 @@ Node *FBXDocument::generate_scene(Ref<GLTFState> p_state, float p_bake_fps, bool
 		root = root->get_owner();
 	}
 	ERR_FAIL_NULL_V(root, nullptr);
+	if (root->get_name() == StringName()) {
```

Comment:
Try is_empty().

Reply:
ok

Reply:
> Try is_empty().


okay i tried it this is stringname and not string so the function is_empty doesnt exist for this
i looked deeper on how gltf does this and i found that yea its done the same way i did it